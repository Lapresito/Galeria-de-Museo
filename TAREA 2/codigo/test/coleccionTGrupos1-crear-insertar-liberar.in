crearColeccionGrupos
liberarColeccionGrupos
crearColeccionGrupos
crearGrupoABB
crearVisitante 4 Rufino Ruiz 22
agregarAGrupoABB
agregarAColeccionGrupos
liberarColeccionGrupos
crearColeccionGrupos
crearGrupoABB
crearVisitante 5 Artemio Gonzalez 24
agregarAGrupoABB
agregarAColeccionGrupos
crearGrupoABB
crearVisitante 6 Esmeralda Gomez 30
agregarAGrupoABB
agregarAColeccionGrupos
liberarColeccionGrupos
Fin
