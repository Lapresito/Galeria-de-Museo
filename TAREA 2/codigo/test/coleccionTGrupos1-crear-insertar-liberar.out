Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> La coleccion de grupos ha sido creado de forma exitosa.
2> Coleccion de grupos liberada
3> La coleccion de grupos ha sido creado de forma exitosa.
4> El grupoABB ha sido creado de forma exitosa.
5> Visitante creadox de forma exitosa.
6> Se ha agregado el visitante al grupo de forma exitosa.
7> Se ha agregado el grupo a la coleccion de forma exitosa.
8> Coleccion de grupos liberada
9> La coleccion de grupos ha sido creado de forma exitosa.
10> El grupoABB ha sido creado de forma exitosa.
11> Visitante creadox de forma exitosa.
12> Se ha agregado el visitante al grupo de forma exitosa.
13> Se ha agregado el grupo a la coleccion de forma exitosa.
14> El grupoABB ha sido creado de forma exitosa.
15> Visitante creadox de forma exitosa.
16> Se ha agregado el visitante al grupo de forma exitosa.
17> Se ha agregado el grupo a la coleccion de forma exitosa.
18> Coleccion de grupos liberada
19> Fin.
